src/main/java/org/gridsuite/shortcircuit/server/entities/FaultResultEntity.java

```diff
@@ -54,6 +56,13 @@ public class FaultResultEntity {
                     columnList = "fault_result_entity_fault_result_uuid")})
     private List<LimitViolationEmbeddable> limitViolations;
 
+    /**
+     * first limit violation of the fault result entity, from 'limitViolations'
+     * Must save it the same table in order to get pageable FaultResultEntity sortable by this limitViolation data
+     */
+    @Embedded
+    private LimitViolationEmbeddable limitViolation;
```

Comment:
Ok for this temporary hack as at the moment, we always get only one limit violation per FaultResult